sdk/appconfiguration/azappconfiguration/client_api_add.go

```diff
@@ -0,0 +1,28 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azappconfiguration
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+type AddConfigurationSettingResponse configurationSettingResponse
+
+type AddConfigurationSettingOptions struct {
+}
+
+func (c *Client) AddConfigurationSetting(ctx context.Context, setting ConfigurationSetting, options *AddConfigurationSettingOptions) (AddConfigurationSettingResponse, error) {
+	etagAny := azcore.ETagAny
+	resp, err := c.appConfigClient.PutKeyValue(ctx, *setting.Key, setting.toGeneratedPutOptions(nil, &etagAny))
+	if err != nil {
+		return AddConfigurationSettingResponse{}, err
+	}
+
+	return (AddConfigurationSettingResponse)(responseFromGeneratedPut(resp)), nil
+}
```

Comment:
Bit of a nit: we try to keep all client methods in the `client.go` file. The response and struct options also are included in this file. It's simpler to have related functions in the same file.

Reply:
I will keep them separated if you don't mind, I find it more convenient to at least  develop in this way - everything related to one method is in one file, and nothing else. Since there aren't that many other entities, it does not clutter the files list.

Reply:
I have moved all methods into a single file now.